src/graphql/queries.js

```diff
@@ -200,6 +200,13 @@ subscription {
     host
     port
     stateTotals
+    latestStateTasks(states: [
+      "failed",
+      "preparing",
+      "submit-failed",
+      "submitted",
+      "running"
+    ])
     taskProxies(sort: { keys: ["cyclePoint"] }) {
```

Comment:
Can we get rid of this `taskProxies` section now?

Also there might be some delay because of the builtin default for `subscription.workflows`:
```
    workflows = List(
        Workflow,
        description=Workflow._meta.description,
        ids=List(ID, default_value=[]),
        exids=List(ID, default_value=[]),
        # TODO: Change these defaults post #3500 in coordination with WUI
        strip_null=Boolean(default_value=False),
        delta_store=Boolean(default_value=False),
        delta_type=String(default_value=DELTA_ADDED),
        initial_burst=Boolean(default_value=True),
        ignore_interval=Float(default_value=2.5),
        resolver=delta_subs
    )
```
So setting the argument `ignoreInterval` to zero would mean all deltas are sent.

Reply:
>Can we get rid of this taskProxies section now?

Yes! It would be pretty bad to merge this one without removing that as the query would still request it :clap:  great catch @dwsutherland !

Reply:
>So setting the argument ignoreInterval to zero would mean all deltas are sent.

Should I change it here, or do you think it could have some performance impact on the backend? If that's the case we can leave it for a follow-up.

Reply:
Removed `taskproxies` from the query.

Reply:
I had it in there as some way of easing the frontend and backend (as any change would trigger delta construction with potentially nested field resolving (i.e. `taskProxies`))

Reply:
Happy to change it to `0` here if it'd make the UI more responsive with little impact on the server. @hjoliver any thoughts on this?

Reply:
Not a lot of impact on the backend for small subscriptions (without querying all nodes), so see how it goes set to `0`